pkg/framework/test/apiserver.go

```diff
@@ -10,35 +10,42 @@ import (
 
 // APIServer knows how to run a kubernetes apiserver.
 type APIServer struct {
-	// URL is the address, a host and a port, the ApiServer should listen on for client connections.
+	// URL is the address the ApiServer should listen on for client connections.
+	//
 	// If this is not specified, we default to a random free port on localhost.
 	URL *url.URL
 
-	// Path is the path to the apiserver binary. If this is left as the empty
-	// string, we will attempt to locate a binary, by checking for the
-	// TEST_ASSET_KUBE_APISERVER environment variable, and the default test
-	// assets directory.
+	// Path is the path to the apiserver binary.
+	//
+	// If this is left as the empty string, we will attempt to locate a binary,
```

Comment:
I'm not sure I understand what's going to happen based on that sentence.

is `TEST_ASSET_KUBE_APISERVER` the name of the binary? I think so since it has the APISERVER name in it.
But then,

> the default test assets directory

what binary do you take there?